configure.ac

```diff
@@ -17,7 +17,7 @@ ERFA_NUMVER
 ## A library supports interfaces from current downto current - age
 ## Revision is the version of the current interface
 ## Follow the instructions in RELEASE.rst to change the version info
-ERFA_LIB_VERSION_INFO(2, 1, 1)
+ERFA_LIB_VERSION_INFO(2, 2, 0)
```

Comment:
Oops, I actually meant (3,0,2) - it's the bullet labeled "else if there is backwards compatible change".

But yeah, I find agree it's quite confusing.  The scheme is a bit old, and comes from libtool, documented in more detail at http://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html .